include/mbedtls/cipher.h

```diff
@@ -682,10 +682,10 @@ int mbedtls_cipher_reset( mbedtls_cipher_context_t *ctx );
  * \param ctx           The generic cipher context. This must be initialized.
  * \param ad            The additional data to use. This must be a readable
  *                      buffer of at least \p ad_len Bytes.
- * \param ad_len        the Length of \p ad Bytes.
+ * \param ad_len        The Length of \p ad Bytes.
```

Comment:
Sorry for nitpicking, but it should be "length" instead of "Length", and "... \p ad _in_ Bytes.".

Reply:
Done.